src/components/application_manager/test/commands/hmi/simple_response_from_hmi_test.cc

```diff
@@ -112,7 +120,7 @@ class ResponseFromHMICommandsTest
   MockEventDispatcher event_dispatcher_;
 
   ResponseFromHMICommandsTest() {
-    ON_CALL(mock_app_manager_, event_dispatcher())
+    ON_CALL(app_mngr_, event_dispatcher())
```

Comment:
@wolfylambova22 Do you use here real instance of application manager? If not - this change is not good. Return it back in all places where mock used.